front/src/pages/item/[id].tsx

```diff
@@ -1,15 +1,37 @@
-import React from "react";
-import ItemView from "@/components/ItemView";
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import ItemView from '../../components/ItemView';
+
+interface RowData {
+  id: string;
+  name: string;
+  age: string;
+  level: string;
+  message: string;
+  important: boolean;
+  status: string;
+}
 
 const ItemPage: React.FC = () => {
-  return (
-    <>
-      <>
-        <ItemView/>
-      </>
-      
-    </>
-  );
-};
+  const [data, setData] = useState<RowData[]>([]);
```

Comment:
the data type is not an array. but one RowData object. it should be set to 
```ts
const [data, setData] = useState<RowData>();
```

that might raise an error due to null initial value that can easily fixed by setting it like this : 
```ts
 const [data, setData] = useState<RowData>({
    id: "",
    name: "",
    age: "",
    level: "",
    message: "",
    important: false,
    status: "",
  });
```